example/lib/main.dart

```diff
@@ -111,6 +115,13 @@ class _MyAppState extends State<MyApp> {
                       frequency: Duration(hours: 1),
                     );
                   }),
+              Text("iOS", style: Theme.of(context).textTheme.headline),
```

Comment:
I think I would definitely emphasise that this is for debugging purposes only.
Even debatable if you really need to expose it to the user. 

If we do opt to have this debugging function. I would check that this only works when `_isInDebugMode` is enabled.